[2023-12-10 20:35:38,667] {processor.py:153} INFO - Started process (PID=13450) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:35:38,667] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:35:38,667] {logging_mixin.py:115} INFO - [2023-12-10 20:35:38,667] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:35:38,677] {logging_mixin.py:115} INFO - [2023-12-10 20:35:38,677] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:35:38,677] {logging_mixin.py:115} INFO - [2023-12-10 20:35:38,677] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:35:38,677] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:35:38,686] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-10 20:36:13,624] {processor.py:153} INFO - Started process (PID=13492) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:36:13,624] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:36:13,625] {logging_mixin.py:115} INFO - [2023-12-10 20:36:13,625] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:36:13,635] {logging_mixin.py:115} INFO - [2023-12-10 20:36:13,634] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:36:13,635] {logging_mixin.py:115} INFO - [2023-12-10 20:36:13,635] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:36:13,635] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:36:13,644] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-10 20:36:46,684] {processor.py:153} INFO - Started process (PID=13529) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:36:46,684] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:36:46,684] {logging_mixin.py:115} INFO - [2023-12-10 20:36:46,684] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:36:46,694] {logging_mixin.py:115} INFO - [2023-12-10 20:36:46,694] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:36:46,694] {logging_mixin.py:115} INFO - [2023-12-10 20:36:46,694] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:36:46,695] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:36:46,703] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-10 20:37:20,385] {processor.py:153} INFO - Started process (PID=13565) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:37:20,385] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:37:20,386] {logging_mixin.py:115} INFO - [2023-12-10 20:37:20,386] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:37:20,396] {logging_mixin.py:115} INFO - [2023-12-10 20:37:20,395] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:37:20,396] {logging_mixin.py:115} INFO - [2023-12-10 20:37:20,396] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:37:20,396] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:37:20,406] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-10 20:37:53,934] {processor.py:153} INFO - Started process (PID=13600) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:37:53,937] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:37:53,937] {logging_mixin.py:115} INFO - [2023-12-10 20:37:53,937] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:37:53,947] {logging_mixin.py:115} INFO - [2023-12-10 20:37:53,947] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:37:53,947] {logging_mixin.py:115} INFO - [2023-12-10 20:37:53,947] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:37:53,947] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:37:53,955] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-10 20:38:27,016] {processor.py:153} INFO - Started process (PID=13635) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:38:27,016] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:38:27,016] {logging_mixin.py:115} INFO - [2023-12-10 20:38:27,016] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:38:27,027] {logging_mixin.py:115} INFO - [2023-12-10 20:38:27,027] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:38:27,027] {logging_mixin.py:115} INFO - [2023-12-10 20:38:27,027] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:38:27,027] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:38:27,040] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-12-10 20:39:01,251] {processor.py:153} INFO - Started process (PID=13683) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:39:01,252] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:39:01,252] {logging_mixin.py:115} INFO - [2023-12-10 20:39:01,252] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:39:01,262] {logging_mixin.py:115} INFO - [2023-12-10 20:39:01,261] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:39:01,262] {logging_mixin.py:115} INFO - [2023-12-10 20:39:01,262] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:39:01,262] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:39:01,273] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-10 20:39:35,238] {processor.py:153} INFO - Started process (PID=13719) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:39:35,238] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:39:35,238] {logging_mixin.py:115} INFO - [2023-12-10 20:39:35,238] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:39:35,249] {logging_mixin.py:115} INFO - [2023-12-10 20:39:35,248] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:39:35,249] {logging_mixin.py:115} INFO - [2023-12-10 20:39:35,249] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:39:35,249] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:39:35,259] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-10 20:40:07,906] {processor.py:153} INFO - Started process (PID=13754) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:40:07,907] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:40:07,908] {logging_mixin.py:115} INFO - [2023-12-10 20:40:07,907] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:40:07,919] {logging_mixin.py:115} INFO - [2023-12-10 20:40:07,918] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:40:07,919] {logging_mixin.py:115} INFO - [2023-12-10 20:40:07,919] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:40:07,919] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:40:07,928] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-12-10 20:40:40,362] {processor.py:153} INFO - Started process (PID=13797) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:40:40,363] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:40:40,363] {logging_mixin.py:115} INFO - [2023-12-10 20:40:40,363] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:40:40,376] {logging_mixin.py:115} INFO - [2023-12-10 20:40:40,375] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:40:40,376] {logging_mixin.py:115} INFO - [2023-12-10 20:40:40,376] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:40:40,376] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:40:40,382] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-10 20:41:13,570] {processor.py:153} INFO - Started process (PID=13831) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:41:13,570] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:41:13,570] {logging_mixin.py:115} INFO - [2023-12-10 20:41:13,570] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:41:13,583] {logging_mixin.py:115} INFO - [2023-12-10 20:41:13,583] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:41:13,584] {logging_mixin.py:115} INFO - [2023-12-10 20:41:13,583] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:41:13,584] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:41:13,590] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-10 20:41:47,035] {processor.py:153} INFO - Started process (PID=13867) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:41:47,035] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:41:47,035] {logging_mixin.py:115} INFO - [2023-12-10 20:41:47,035] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:41:47,048] {logging_mixin.py:115} INFO - [2023-12-10 20:41:47,047] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:41:47,048] {logging_mixin.py:115} INFO - [2023-12-10 20:41:47,048] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:41:47,048] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:41:47,054] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-10 20:42:21,504] {processor.py:153} INFO - Started process (PID=13902) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:42:21,505] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:42:21,505] {logging_mixin.py:115} INFO - [2023-12-10 20:42:21,505] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:42:21,518] {logging_mixin.py:115} INFO - [2023-12-10 20:42:21,517] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:42:21,518] {logging_mixin.py:115} INFO - [2023-12-10 20:42:21,518] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:42:21,518] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:42:21,525] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-10 20:42:55,197] {processor.py:153} INFO - Started process (PID=13938) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:42:55,198] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:42:55,199] {logging_mixin.py:115} INFO - [2023-12-10 20:42:55,198] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:42:55,213] {logging_mixin.py:115} INFO - [2023-12-10 20:42:55,213] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:42:55,214] {logging_mixin.py:115} INFO - [2023-12-10 20:42:55,213] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:42:55,214] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:42:55,220] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-12-10 20:43:27,441] {processor.py:153} INFO - Started process (PID=13975) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:43:27,441] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:43:27,442] {logging_mixin.py:115} INFO - [2023-12-10 20:43:27,441] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:43:27,455] {logging_mixin.py:115} INFO - [2023-12-10 20:43:27,455] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:43:27,455] {logging_mixin.py:115} INFO - [2023-12-10 20:43:27,455] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:43:27,455] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:43:27,461] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-10 20:44:00,956] {processor.py:153} INFO - Started process (PID=14012) to work on /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:44:00,957] {processor.py:641} INFO - Processing file /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-10 20:44:00,957] {logging_mixin.py:115} INFO - [2023-12-10 20:44:00,957] {dagbag.py:507} INFO - Filling up the DagBag from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:44:00,969] {logging_mixin.py:115} INFO - [2023-12-10 20:44:00,969] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-10 20:44:00,969] {logging_mixin.py:115} INFO - [2023-12-10 20:44:00,969] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-10 20:44:00,969] {processor.py:653} WARNING - No viable dags retrieved from /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-10 20:44:00,975] {processor.py:161} INFO - Processing /home/vmac/lab3/lab3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
